### ISSUE TITLE
Touch Mappings port to Override Size repos

### ISSUE THREAD

**Issue by srevinsaju** [srevinsaju/guiscrcpy#74]
python3 -V : Python 3.8.2
python -V : Command 'python' not found
Btw, using python3 scripts/guiscrcpy-mapper i am able to see the key mapper and it generate a json file called guiscrcpy.mapper.json on .config/guiscrcpy so for me i can start prepare a file to my need but i need an example.
using the guiscrcpy mapper i have this message error:
`Lets Check
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/guiscrcpy-3.3-py3.8.egg/guiscrcpy/lib/mapper.py", line 185, in keyreg
    fixy = rely * int(dimValues[1])
ValueError: invalid literal for int() with base 10: '2960\nOverride size'
PyQt5.QtCore.QPoint(365, 355) LAST`
That why i need and example file to generate manually if we are in need.
Thank :+1:

_Originally posted by @karousn in https://github.com/srevinsaju/guiscrcpy/issues/13#issuecomment-598792892_

**Comment by karousn** [comment]
any news about this ?

**Comment by srevinsaju** [comment]
@karousn Maybe I will finish that by this weekend. I apologize for the delay

**Comment by srevinsaju** [comment]
@karousn I fixed the mapper, can you please test it?

**Comment by karousn** [comment]
The last tag i can get it is 3.4 so i try it and i have this message:
Waiting for device
Device : OK!
['1440', '2960\nOverride size']  DIMENSIONS 
LOG: Key:Pos JSON Configuration file found in  /home/user/.config/guiscrcpy/  directory
KEY:A {'key': [], 'pos': []}
[SERVER] LISTENING VALUES: Your keys are being listened by server. 
{'key': [], 'pos': []}

**Comment by srevinsaju** [comment]
@karousn can you try out https://github.com/srevinsaju/guiscrcpy/commit/2583d1f3d59295fe7d7e49f98ccaf1d76bb2fa70

**Comment by karousn** [comment]
To do the test i do:
```
git checkout 2583d1f
sudo pip3 uninstall guiscrcpy
python3 setup.py install --user
```
When i try to use guiscrcpy directly in any folder i have  `ModuleNotFoundError: No module named 'PyQt5.sip'`  as message error so i use `python -m guiscrcpy` or `python scripts/guiscrcpy`
before that i activate the Pointer location on my phone to see the coordination of my click, and i modify the json keymapper file directly and i restart my guiscrcpy after that in each modification.( it was crashing some time.. i use force quit to exit).
i see that you use `adb shell input tap X Y` to send the event of click to my phone, it is a little bit slow specially in gaming. if you have any idea how can i improve that.

**Comment by srevinsaju** [comment]
> i see that you use adb shell input tap X Y to send the event of click to my phone, it is a little bit slow specially in gaming. if you have any idea how can i improve that.
I will have to use scrcpy's fast adb shellinput's but i have 0 Java background, am still 17 lol

```bash
pip3 install PyQt5-sip --user
rm ~/.config/guiscrcpy/*mapper*
guiscrcpy-mapper
guiscrcpy
```
Hope this would work
You can check
```
guiscrcpy-mapper --help
```
to see the possible options.
Every time you remove the json file, the mapper is reseted to the original values

**Comment by karousn** [comment]
by curiosity witch lib java you will use to accelerate the send of the event action to the phone ?

**Comment by srevinsaju** [comment]
I have to use scrcpy's scrcpy-server Input.java to create a socket to pass she'll input events. The current adb shell input does a lot of checos, for eg: check if device is connected, copy the process to mobile, start an Android server daemon, and then run the process and then wait for the process to finish on Android and then send the complete signal to the PC. This is quite complicated and takes approxately 1.5 seconds. Rather, only one server can be started, and then only the input processes can be sent over the tunnel. I am bad at networking and Java, so it's quite complicated to understand Java or Sockets. If you have any clue, I would be glad to use it.


See Scrcpy's issue https://github.com/Genymobile/scrcpy/issues/231#issuecomment-414111753

**Comment by srevinsaju** [comment]
Closed by [3.5](https://github.com/srevinsaju/guiscrcpy/releases/tag/3.5)
